en/Extending Obsidian/Obsidian URI.md

```diff
@@ -21,140 +30,239 @@ The `action` parameter is the action that you would like to perform. Available a
 - `daily` to create or open your daily note.
 - `search` to open a search.
 
-
 > [!warning] Encoding
 > Ensure that your values are properly URI encoded. For example, forward slash characters `/` must be encoded as `%2F` and space characters must be encoded as `%20`.
-> 
- This is especially important because an improperly encoded "reserved" character may break the interpretation of the URI. [See here for details](https://en.wikipedia.org/wiki/Percent-encoding).
+>
+> This is especially important because an improperly encoded "reserved" character may break the interpretation of the URI. [See here for details](https://en.wikipedia.org/wiki/Percent-encoding).
+
+## Get an Obsidian URL
+
+To get an Obsidian URL for any note, use the **Copy Obsidian URL** command. This command is available in:
+
+- The [[Command palette]]
+- The three-dot menu in the note header
+- The right-click context menu for files in the file explorer
+
+The command copies an `obsidian://open` URL to your clipboard that points to the note.
+
+## Common parameters
+
+The following parameters are used across multiple URI actions. Understanding these common parameters will help you construct URIs for different actions more easily.
+
+### `vault`
+
+Specifies which vault to target. Can be either the vault name or the vault ID[^1].
+
+- **Vault name**: The human-readable name of your vault (e.g., `my vault`).
+- **Vault ID**: A unique 16-character identifier for the vault (e.g., `ef6ca3e3b524d22f`). Using the vault ID is more reliable if you have multiple vaults with similar names or if you rename your vault.
+
+### `file`
+
+Specifies which file to target within the vault. Can be either a file name or a path from the vault root to the specified file.
+
+- If the file extension is `md`, the extension can be omitted.
+- For files in subdirectories, use forward slashes to specify the path (e.g., `folder/subfolder/note`).
+
+### `path`
+
+An absolute file system path to a file on your computer.
+
+- Using this parameter will override both `vault` and `file`.
+- The app will search for the most specific vault that contains the specified file path.
+- The remainder of the path after the vault root replaces the `file` parameter.
+
+For example, if you have a vault at `/home/user/my vault` and use `path=/home/user/my vault/folder/note`, this is equivalent to setting `vault=my vault` and `file=folder/note`.
 
 ## Open note
 
 The `open` action opens an Obsidian vault, or opens a file within that vault.
 
+### Parameters
+
+The `open` action uses the [[#Common parameters]] (`vault`, `file`, `path`).
```

Comment:
nit: maybe nicer to have a display that isn't capitalized for this link? Unsure what the help site does elsewhere, so feel free to ignore.